$NetBSD: patch-Makefile,v 1.2 2023/09/10 10:18:44 yhardy Exp $

Let pkgsrc determine the search paths, instead of forcing
the prefix.

--- Makefile.orig	2022-12-09 11:37:27.000000000 +0000
+++ Makefile
@@ -286,7 +286,7 @@ INCLUDE ?= -I/usr/include
 else ifeq ($(YQ2_OSTYPE),FreeBSD)
 INCLUDE ?= -I/usr/local/include
 else ifeq ($(YQ2_OSTYPE),NetBSD)
-INCLUDE ?= -I/usr/X11R7/include -I/usr/pkg/include
+INCLUDE ?= -I/usr/X11R7/include
 else ifeq ($(YQ2_OSTYPE),OpenBSD)
 INCLUDE ?= -I/usr/local/include
 else ifeq ($(YQ2_OSTYPE),Windows)
@@ -301,7 +301,7 @@ LDFLAGS ?= -L/usr/lib
 else ifeq ($(YQ2_OSTYPE),FreeBSD)
 LDFLAGS ?= -L/usr/local/lib
 else ifeq ($(YQ2_OSTYPE),NetBSD)
-LDFLAGS ?= -L/usr/X11R7/lib -Wl,-R/usr/X11R7/lib -L/usr/pkg/lib -Wl,-R/usr/pkg/lib
+LDFLAGS ?= -L/usr/X11R7/lib
 else ifeq ($(YQ2_OSTYPE),OpenBSD)
 LDFLAGS ?= -L/usr/local/lib
 else ifeq ($(YQ2_OSTYPE),Windows)
